logstash-output-http.gemspec

```diff
@@ -1,7 +1,7 @@
 Gem::Specification.new do |s|
 
   s.name            = 'logstash-output-http'
-  s.version         = '3.1.1'
+  s.version         = '4.0.0'
```

Comment:
I would just make it 3.2.0 since an older config is compatible with new changes. We're only adding new options. We've changed the internal behavior, but the config remains the same.